src/python/essentia/pytools/extractors/melspectrogram.py

```diff
@@ -0,0 +1,229 @@
+# Copyright (C) 2006-2020  Music Technology Group - Universitat Pompeu Fabra
+#
+# This file is part of Essentia
+#
+# Essentia is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Affero General Public License as published by the Free
+# Software Foundation (FSF), either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the Affero GNU General Public License
+# version 3 along with this program. If not, see http://www.gnu.org/licenses/
+
+import argparse
+import os
+
+import numpy as np
+
+from essentia import Pool
+from essentia import run
+
+from essentia.streaming import (MonoLoader, FrameCutter, Windowing, Spectrum,
+                                MelBands, UnaryOperator)
+
+ZERO_PADDING = 0
+WINDOW_TYPE = 'hann'
+FRAME_SIZE = 1024
+HOP_SIZE = 512
+NUMBER_BANDS = 24
+SAMPLE_RATE = 44100.
+LOW_FREQUENCY_BOUND = 0.
+HIGH_FREQUENCY_BOUND = 22050.
+WARPING_FORMULA = 'htkMel'
+WEIGHTING = 'warping'
+NORMALIZE = 'unit_sum'
+BANDS_TYPE = 'power'
+COMPRESSION_TYPE = 'shift_scale_log'
+
+
+def melspectrogram(filename, npy_file=None, force=False, verbose=False, sample_rate=SAMPLE_RATE, frame_size=FRAME_SIZE,
+    hop_size=HOP_SIZE, window_type=WINDOW_TYPE, zero_padding=ZERO_PADDING, low_frequency_bound=LOW_FREQUENCY_BOUND,
+    high_frequency_bound=HIGH_FREQUENCY_BOUND, number_bands=NUMBER_BANDS, warping_formula=WARPING_FORMULA,
+    weighting=WEIGHTING, normalize=NORMALIZE, bands_type=BANDS_TYPE, compression_type=COMPRESSION_TYPE):
+    """Computes the mel spectrogram given the audio filename.
+    When the parameter `npy_file` is specified, the data is saved to disk as a numpy array (.npy).
+    Use the parameter `force` to overwrite the numpy array in case it already exists.
+    The rest of parameters are directly mapped to Essentia algorithms as explained below.
+
+    Note: this functionallity is also avaiable as a command line script.
```

Comment:
```suggestion
    Note: this functionality is also available as a command line script.
```

Do you also want to explain how to call this commandline script? Where will people be seeing this comment?